packages/firestore/test/lite/integration.test.ts

```diff
@@ -2116,6 +2117,45 @@ describe('countQuery()', () => {
     });
   });
 
+  it('count with filter and order by', () => {
+    return withTestCollectionAndInitialData(testDocs, async collection => {
+      const query1 = query(
+        collection,
+        where('author', '==', 'authorB'),
```

Comment:
Please remove the filter from this test case. The test should be focused on order-by and adding a filter is not relevant and distracts from what the test is actually testing.